jobserv/flask.py

```diff
@@ -59,6 +60,14 @@ def create_app(settings_object="jobserv.settings"):
     app.wsgi_app = ProxyFix(app.wsgi_app)
     app.config.from_object(settings_object)
 
+    # json_logging can only be initialized *once*. When running with gunicorn,
+    # this gets called a couple times.
+    if not getattr(create_app, "__logging_hack_initialized", None):
+        json_logging.init_flask(enable_json=True)
+        json_logging.init_request_instrument(app)
+        json_logging.config_root_logger()
+        create_app.__logging_hack_initialized = True
```

Comment:
It is quite strange that `create_app` is called several times; but maybe that's the case.

Anyway, even in this case I believe `json_logging.init_request_instrument(app)` needs to be called for every app. Otherwise, we'd have a part of `app` objects half-initialized.